### PR TITLE
amir.test.js file was added and testing was completed

### PR DESCRIPTION
**Pull request by amirmammadov**

### CODE REVIEW

File: index.js
Comment (elnurastanov) [inline index.js:1]: Revert changes for this file

File: elnur.test.js
Comment (elnurastanov) [inline elnur.test.js:1]: Undo deletion for this file

**Review by elnurastanov — CHANGES_REQUESTED** [review]

**Comment by amirmammadov** [comment]
I did them

File: amir.test.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+});
+
+describe("getLocalCurrency function", () => {
+  it("should return correct currency based on country code", () => {
+    expect(getLocalCurrency("AZ")).toBe("AZN");
+    expect(getLocalCurrency("TR")).toBe("TL");
+    expect(getLocalCurrency("US")).toBe("US");
+    expect(getLocalCurrency("PL")).toBe("ZL");
+    expect(getLocalCurrency("GE")).toBe("EU");
+  });
```
Comment (elnurastanov) [inline amir.test.js:127]: Where is not found case?

**Review by elnurastanov — CHANGES_REQUESTED** [review]

**Review by elnurastanov — APPROVED** [review]

**Pull request merged**